bitnami/harbor/templates/_helpers.tpl

```diff
@@ -439,6 +443,13 @@ Return the proper Harbor Core image name
 {{- include "common.images.image" ( dict "imageRoot" .Values.core.image "global" .Values.global ) -}}
 {{- end -}}
 
+{{/*
+Return the proper Harbor Exporter image name
+*/}}
+{{- define "harbor.exporter.image" -}}
+{{- include "common.images.image" ( dict "imageRoot" .Values.exporter.image "global" .Values.global ) -}}
+{{- end -}}
```

Comment:
Nice! Please also update "harbor.imagePullSecrets" (line 541) including this image in the list.